libraries/core/src/main/java/com/paritytrading/philadelphia/FIXCheckSums.java

```diff
@@ -3,24 +3,8 @@
 import java.nio.BufferUnderflowException;
 import java.nio.ByteBuffer;
 
-/**
- * Utilities for working with checksums.
- */
-public class FIXCheckSums {
+class FIXCheckSums {
 
-    private FIXCheckSums() {
-    }
-
-    /**
-     * Return the sum of bytes in a buffer.
-     *
-     * @param buffer a buffer
-     * @param offset the offset
-     * @param length the length
-     * @return the sum of bytes in the buffer
-     * @throws BufferUnderflowException if the length from the offset exceeds
-     *   the limit of the buffer
-     */
     public static long sum(ByteBuffer buffer, int offset, int length) {
```

Comment:
Could be package private?

Reply:
Yes, indeed. There are a few other places that have superfluous `public` modifiers within package-private classes. I thought about going through and cleaning these up in one go.